lib/scheduler.js

```diff
@@ -0,0 +1,39 @@
+var Promise = require('bluebird')
+
+module.exports = function(service, options) {
+  var pollingInterval = options.schedulePollingInterval || 5000
+  var stopped = false
+
+  this.getDueMessages = function*() {
+    return yield service.storageTransport.getDueMessages(service, new Date().getTime())
```

Comment:
You should replace `new Date().getTime()` with the static Date method meant for that: `Date.now()`